.evergreen/legacy_config_generator/evergreen_config_lib/variants.py

```diff
@@ -422,25 +422,25 @@ def days(n: int) -> int:
         ],
         {"CC": "clang"},
     ),
+    # Test with Ubuntu 20.04. MongoDB 7.0 no longer ships binaries for Ubuntu 18.04. AWS setup scripts expect Ubuntu 20.04+. MongoDB 4.4 and 5.0 are not available on Ubuntu 20.04.
```

Comment:
Confused by this comment vs. matrix below. Should it be something like the following instead?

```
# AWS setup scripts expect Ubuntu 20.04+. MongoDB 4.4 and 5.0 are not available on 22.04.
Variant("aws-ubuntu2004", ...),

# AWS setup scripts expect Ubuntu 20.04+. MongoDB 7.0 and newer are not available on Ubuntu 22.04.
Variant("aws-ubuntu2204", ...),
```